.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 # aws-doc-sdk-examples Pull Request
```

Comment:
Since we're providing general guidelines at the top, I'm assuming that the requirements for each section below are minimum requirements? I've added edits on this assumption.

Reply:
Yes, minimum requirements to the extent that I understand them.